types/react-native-auth0/index.d.ts

```diff
@@ -249,6 +249,10 @@ export interface ClearSessionParams {
     customScheme?: string;
 }
 
+export interface ClearSessionOptions {
```

Comment:
Since you're updating the definition, can you please put all 4 available options described at https://auth0.github.io/react-native-auth0/WebAuth.html?

Reply:
@marknelissen this refers to the `clearSession` method and not the `authorize` method. `clearSession` only has 1 option - https://auth0.github.io/react-native-auth0/WebAuth.html#clearSession